pom.xml

```diff
@@ -247,7 +247,8 @@
                             <target name="ktlint">
                                 <java taskname="ktlint" dir="${basedir}" fork="true" failonerror="true"
                                       classpathref="maven.plugin.classpath" classname="com.pinterest.ktlint.Main">
-                                    <arg value="src/**/*.kt"/>
+                                    <arg value="src/main/**/*.kt"/>
```

Comment:
could you please change here to src/main/kotlin

Reply:
Done.